src/components/loader/loader.component.tsx

```diff
@@ -12,14 +12,15 @@ import StyledLoaderSquare, {
 } from "./loader-square.style";
 
 export interface LoaderProps
-  extends StyledLoaderSquareProps,
+  extends Omit<StyledLoaderSquareProps, "loaderNumber">,
     MarginProps,
     TagProps {
   /** Specify a custom accessible name for the Loader component */
   "aria-label"?: string;
 }
 
 export const Loader = ({
+  color = "default",
```

Comment:
Comment: I think we should use `variant` here so we're consistent with how we've done it in most of our other newer components